run-test

```diff
@@ -1,13 +1,35 @@
 #!/bin/bash -e
+# For coverage tests, run as
+#     coverage erase
+#     PATH=$PWD:$PATH RUNNER="coverage run --include=$PWD/**/*" ./run-test
+#     coverage html
+# See ./run-coverage
```

Comment:
Well, this lines are slightly different from the ones in `run-coverage` and I would argue that running tests is more frequent what users need than to run the coverage. Maybe explain these two use cases just shortly in two lines, e.g.
```bash
# For running the tests, use `./run-test`
# For coverage tests, use `./run-coverage` which will use these tests here
```